### DISCUSSION TITLE
When are packages released to npm?

### DISCUSSION THREAD

**Discussion by mehradrafigh** [OfficeDev/microsoft-365-agents-toolkit#11533] [General]
In particular, I am interested to understand when packages are published to npm
https://github.com/OfficeDev/TeamsFx/issues/11206 was fixed a while ago, but the package still needs to be updated on npm.

**Comment by MuyangAmigo** [comment]
Hey @mehradrafigh  - There is a new version published a couple days ago: https://www.npmjs.com/package/@microsoft/adaptivecards-tools/v/1.3.4
